app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -18,7 +18,25 @@ def textual_group_relationships
   end
 
   def textual_group_status
-    %i(refresh_status)
+    textual_authentications + %i(refresh_status)
+  end
+
+  def textual_authentications
+    authentications = @ems.authentications
+    return [{:label => _("Default Authentication"), :title => t = _("None"), :value => t}] if authentications.blank?
+
+    authentications.collect do |auth|
+      label =
+        case auth.authtype
+        when "default" then _("Default")
+        when "bearer" then _("Bearer")
```

Comment:
Nope. This won't fly with gettext. You cannot do any expressions inside a gettext call.
Although it would technically work, the thing inside the gettext would not be extracted
during collection of strings for translation (it's a dynamically created strings, so there's
nothing to collect).